runtime/flang/fpcvt.c

```diff
@@ -31,6 +31,17 @@ union ieee {
   int i[2];
 };
 
+union ieeeq {
+  long double q;
+  struct {
+    unsigned int lm[3];
+    unsigned int hm : 16;
+    unsigned int e : 15;
+    unsigned int s : 1;
+  } v;
+  int i[4];
```

Comment:
C11;s limits.h  defines LDBL_* preproc definitions that could be used to determine the real number of ints required here.

Reply:
I couldn't find the preprocessor definitions you alluded to; could you clarify what you meant? Would something like `int i[sizeof(long double)/sizeof(int)]` be better? The `int i[4]` declaration is modeled after the pre-existing `union ieee` just a few lines above. Do we need to change that too?

Reply:
I would make it consistent if it isn't too much of the hassle.

Reply:
OK I have changed these declarations to calculate the number of array elements with a division of type sizes.